metrics/metrics.go

```diff
@@ -203,7 +203,7 @@ var DefaultViews = []*view.View{
 	{
 		Measure:     PersistDuration,
 		Aggregation: defaultMillisecondsDistribution,
-		TagKeys:     []tag.Key{TaskType, Table, ActorCode},
+		TagKeys:     []tag.Key{TaskType, Table},
```

Comment:
Oh, why is ActorCode removed?

Reply:
High cardinality, maybe?

Reply:
When I check the metric and code, `ActorCode` was never included. So I'm removing it from TagKeys.

Reply:
Ohh its probably because we forget to include it when logging the metric...

Reply:
I see. I think the cardinality is already high enough :P